zerver/migrations/0029_realmalias_realm_index.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0028_userprofile_tos_version'),
+    ]
```

Comment:
This migration should create RealmAlias objects for all existing realms too, right?